lib/Parser/Parse.cpp

```diff
@@ -10868,7 +10868,7 @@ ParseNodePtr Parser::Parse(LPCUTF8 pszSrc, size_t offset, size_t length, charcou
 
     m_scriptContext->AddSourceSize(m_length);
 
-    if(!m_parseType != ParseType_Deferred)
+    if (m_parseType != ParseType_Deferred)
```

Comment:
Yeah, I think so- I think that was a typo there (we check for m_parseType != ParseType_Deferred when we fire the start event)